casbin/config/config.cpp

```diff
@@ -0,0 +1,186 @@
+#include "config.h"
+#include "../util/util.h"
+#include <iostream>
+#include <sstream>
+#include <algorithm>
+#include <fstream>
+
+const string DEFAULT_SECTION = "default";
+const string DEFAULT_COMMENT = "#";
+const string DEFAULT_COMMENT_SEM = ";";
+const string DEFAULT_MULTI_LINE_SEPARATOR = "\\";
+
+
+
+Config::Config() {
+
+}
+
+Config* Config::NewConfigFromFile(Error& err,const string& path) {

```

Comment:
I think return a `Config` or `Config&` is much better, raw pointers are very dangerous.

Reply:
Yes,you are right.But if we return a Config&,when the function done,the resource will be released.
I am thinking a better way to deal with the pointer issue.